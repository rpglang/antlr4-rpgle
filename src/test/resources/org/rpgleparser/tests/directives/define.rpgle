      /DEFINE def1
      /UNDEFINE def1    