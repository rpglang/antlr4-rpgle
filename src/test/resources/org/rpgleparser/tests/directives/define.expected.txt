/*===TOKENS===*/
HIDDEN:DIRECTIVE         < />
HIDDEN:DIR_DEFINE        <DEFINE>
HIDDEN:DIR_OtherText     <def1>
DIRECTIVE                < />
DIR_UNDEFINE             <UNDEFINE>
DIR_OtherText            <def1>
/*===TREE===*/
(r (statement (directive ' /' (dir_undefine UNDEFINE def1) <EOF>)))
/*======*/